### ISSUE TITLE
Batchnormalization is incorrect with cpu backend and channelsFirst data.

### ISSUE THREAD

**Issue by frpays** [tensorflow/tfjs#1106]
This issue has been checked in tf 14.1 and 1.3.3.

All browsers are impacted, specifically Chrome 71.

You can reproduce it in https://github.com/frpays/lc0-js.
In the Network constructor, force the cpu backend and the dataformat.

change 

```
    var format = 'channelsFirst';
    if (this.backend == 'cpu') format = 'channelsLast';
    this.setDataFormat(format);
```

with 

```
    tf.setBackend(('cpu');
    this.setDataFormat('channelsFirst');
```

**Comment by frpays** [comment]
Preliminary tests showed that, with the cpu backend, the batchnormalization treats everything as channelsLast, ignoring the dataFormat directive.

**Comment by rthadur** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
